src/Shared/Recruit.Vacancies.Client/Recruit.Vacancies.Client.csproj

```diff
@@ -20,7 +20,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="CsvHelper" Version="12.1.2" />
-    <PackageReference Include="HtmlSanitizer" Version="4.0.205" />
+    <PackageReference Include="HtmlSanitizer" Version="4.0.210" />
```

Comment:
QA should explicitly regression test all these package updates

Reply:
Yes